Assets/Scripts/UI/Leaderboard/UILeaderboardMenu.cs

```diff
@@ -102,6 +125,11 @@ private void QueryRanksCompleted(Result result)
             if(result != Result.Success)
             {
                 Debug.LogErrorFormat("UILeaderboardMenu (QueryRanksCompleted): returned result error: {0}", result);
+
+                // Even if we failed, try to refresh the leaderboard again later
+                if (refreshLeaderboardCoroutine != null) this.StopCoroutine(refreshLeaderboardCoroutine);
```

Comment:
With the exception of `continue` statements within loops, I believe our convention for conditionals is to always put them on new-lines with curly brackets on lines of their own.

Reply:
That's a really agreeable standard. I'll change this over now.